pyvene/models/blip/modelings_intervenable_blip_itm.py

```diff
@@ -0,0 +1,96 @@
+"""
+Each modeling file in this library is a mapping between
+abstract naming of intervention anchor points and actual
+model module defined in the huggingface library.
+
+We also want to let the intervention library know how to
+config the dimensions of intervention based on model config
+defined in the huggingface library.
+"""
+
+from ..constants import *
+
+"""blip ITM base model"""
+blip_itm_type_to_module_mapping = {
+    # TODO: not sure why these are commented out in the BlipVQA implementation
```

Comment:
we can uncomment these if Zen confirms that arbitrary key strings are good now

Reply:
yes, i think arbitrary strings are good! please test them out though